contracts/DFRewards.sol

```diff
@@ -114,6 +109,15 @@ contract DFRewards is Ownable, ReentrancyGuard {
         IERC20(_token).transfer(msg.sender, amount);
     }
 
+    function addStrategy(address _strategy) external onlyOwner {
+        // maybe add a timeout here
```

Comment:
Why "maybe add a timeout"? 

I suggest: either do it, or don't, but don't comment with "maybe do x"

Reply:
I'm in favor of the first one too.